scripts/validate_docstrings.py

```diff
@@ -471,6 +471,46 @@ def parameter_desc(self, param):
                 desc = desc[:desc.index(full_directive)]
         return desc
 
+    def parameter_desc_list(self, param):
+        """Return a parameter description as a list.
+
+        Each line of the parameter docstring is a string in the list.
+        """
+        i = list(self.doc_parameters).index(param)
```

Comment:
Rather than converting the dict to a list and grabbing an index from there can you not just `filter` self.doc_parsed_data (list of named tuples) for the method you want?